install.sh

```diff
@@ -103,7 +103,8 @@ fi
 su -c "cp $SERVER_PATH/scripts/units/fbp-runner@.service $SYSTEMD_SERVICE_PATH/ &&
        cp $SERVER_PATH/scripts/units/soletta-dev-app-server.service.in $SYSTEMD_SERVICE_PATH/soletta-dev-app-server.service &&
        sed -i "s@PATH@"$SERVER_PATH"@" $SYSTEMD_SERVICE_PATH/soletta-dev-app-server.service &&
-       sed -i "s@"NODE_BIN_NAME"@"$NODE_BIN_NAME"@" $SYSTEMD_SERVICE_PATH/soletta-dev-app-server.service"
+       sed -i "s@"NODE_BIN_NAME"@"$NODE_BIN_NAME"@" $SYSTEMD_SERVICE_PATH/soletta-dev-app-server.service &&
+       sed -i "s@"/usr/bin/sol-fbp-runner"@"$FBP_RUNNER_PATH"@" $SYSTEMD_SERVICE_PATH/fbp-runner@.service"
```

Comment:
where it is (/usr/bin/sol-fbp-runner) ? Should you change it there ? Then this sed will not be necessary